setup.py

```diff
@@ -29,7 +29,7 @@
         # TODO: can we safely figure out how to use a pair of options
         # like these, adapted to as many platforms as possible, to use
         # multiple processors when available?
-        # extra_compile_args=['-fopenmp'],
+        extra_compile_args=['-ffloat-store'],
         # extra_link_args=['-fopenmp'],
```

Comment:
Could you keep the `openmp` compile arg next to the comment too? If I ever return to the attempt to do parallel processing, both those lines will need to be uncommented together. Thanks!

Reply:
Sure, my fault. Should have kept that in.